rpc/block_svc_routines.c

```diff
@@ -3961,6 +3967,8 @@ block_create_cli_1_svc_st(blockCreateCli *blk, struct svc_req *rqstp)
   cobj.rb_size = blk->rb_size;
   GB_STRCPYSTATIC(cobj.gbid, gbid);
   GB_STRDUP(cobj.block_hosts,  blk->block_hosts);
+  cobj.xdata.xdata_len = HOST_NAME_MAX;
```

Comment:
I guess this should be actually 'strlen(gbConf.volServer)'; ?

Reply:
Done